doc/nrf/releases/release-notes-1.4.1.rst

```diff
@@ -0,0 +1,109 @@
+.. _ncs_release_notes_141:
+
+|NCS| v1.4.1 Release Notes
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
+It includes the MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.
+
+The |NCS| is where you begin building low-power wireless applications with Nordic Semiconductor nRF52, nRF53, and nRF91 Series devices.
+nRF53 Series devices (which are pre-production) and Zigbee and Bluetooth mesh protocols are supported for development in v1.4.1 for prototyping and evaluation.
+Support for production and deployment in end products is coming soon.
+
+
+Highlights
+**********
+
+* Qualified Zephyr Host to be conformant with Bluetooth Core Specification v5.2
```

Comment:
"Zephyr Host" should be "BLE Host Subsystem" in all cases

Reply:
Updated using "Bluetooth Host" instead of "BLE Host" after checking with @joerchan .